frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,21 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
```

Comment:
I can't this working locally but I recall this wasn't needed. Did you test without it?

Reply:
Also same class naming comment as below.
`kv-create-vm__virtual-hardware__wrapper` very nitpicky I know, but shorter names are easier to scan.